packages/forms/src/directives/validators.ts

```diff
@@ -237,18 +259,18 @@ export const MIN_VALIDATOR: StaticProvider = {
   selector:
       'input[type=number][min][formControlName],input[type=number][min][formControl],input[type=number][min][ngModel]',
   providers: [MIN_VALIDATOR],
-  host: {'[attr.min]': 'min ?? null'}
+  host: {'[attr.min]': 'enabled() ? min : null'}
 })
 export class MinValidator extends AbstractValidatorDirective implements OnChanges {
   /**
    * @description
    * Tracks changes to the min bound to this directive.
    */
-  @Input() min!: string|number;
+  @Input() min!: string|number|null;
   /** @internal */
   override inputName = 'min';
   /** @internal */
-  override normalizeInput = (input: string): number => parseFloat(input);
+  override normalizeInput = (input: string|number): number => toInteger(input);
```

Comment:
I think it should be `toFloat`, not `toInteger`:
```suggestion
  override normalizeInput = (input: string|number): number => toFloat(input);
```
Btw, it's interesting that we don't have failing tests after this change. @iRealNirmal if you get a chance, could you please add a test that would verify that we can not have a regression here (we can probably have min/max and a value in tests as float numbers and come up with a case when `parseInt` would cause an issue)?

Reply:
hi @AndrewKushnir thanks for catching mistake. Regarding your comment
`we can probably have min/max and a value in tests as float numbers and come up with a case when parseInt would cause an issue`

So I doubt this can be valid scenario because if min is `1.0` and max is `3.0` and user enters just `2` then still it will be correct to user always enter decimal value developers should add pattern or regex.

Please feel free tp correct me If am wrong in above scenario or in case you have other thought in your mind.

Reply:
I was thinking of a scenario where `min="1.5"` (thus `min` becomes `1` after `parseInt` operation) and you try to input `1.6`. With the `parseFloat` this would work, but with the `parseInt` call (which you had in the PR for some time) - validation would not indicate this as a problem. Please let me know if you need more information.

Reply:
You are correct, It's strange what  I saw that we already have test  cases of float number under
`should validate max for float number` but it didn't failed when I changed it toInteger also  I wrote some test cases but still it worked. I have to spend some  more time to find a root cause and fix for this. Feel Free to let me know in case you have something in mind. In case I found some fix or root cause I will update this  thread.

Reply:
@AndrewKushnir reason of it's not failing is in validator method of
maxValidator we are using parseFloat inside file (form/src/validators.ts) let me know what you expect me  to do about it.